acceptance/README.md

```diff
@@ -1,257 +1,467 @@
-
-    HOST_NAMES=lvwwr9tdplg351u bundle exec rake ci:release_hosts
-    HOST_NAMES=lvwwr9tdplg351u,ylrqjh5l6xvym4t bundle exec rake ci:release_hosts
+Always clean up after yourself
+```
+bundle exec beaker destroy
+```
 
-### Syncing local development code with VMPooler Windows machines
+-------------
 
-If you want to copy local code to a Windows machine that is running in beaker, you can use the `ci:sync:windows` rake task:
+## Configuration
+### Environment Variables
+A detailed description of the available environment variables can be found by
+running `bundle exec rake ci:help`. This will print a list of both required and
+optional environment variable with short descriptions on how they are used.
+Please review all of these options as they will impact how your test servers
+are provisioned. This rake task is the most up to date source for this
+information.
+
+### Customizing Test Targets
+If you are using the vmpooler hypervisor and the internal pooling api, you can
+customize the platforms to test on using the HOSTS environment variable. The
+internal pooling api is only available to Puppet, Inc. employees. You'll set the
+`HOSTS` environment variable to the host string you want to test, such as
+`HOSTS=redhat7-64ma-windows2012r2-64a`.
+
+If you are unsure of the syntax you need, you can verify the host string against
+the options in
+[beaker hostgenerator](https://github.com/puppetlabs/beaker-hostgenerator/blob/master/lib/beaker-hostgenerator/data.rb). Be sure to use the key from the hash structure.
+You can safely ignore the data associated with the key.
+Generally, this string will be in the format
+`{platform}{version}-{architecture}{role/s}`. You will most often use either the
+agent (a) or master (m) role, but you can find a list of available roles in
+[beaker hostgenerator](https://github.com/puppetlabs/beaker-hostgenerator/blob/master/lib/beaker-hostgenerator/roles.rb).
+Multiple hosts in the string are separated with a dash(`-`). You must have at
+least one agent and at least one master.
+
+Be careful not to confuse the different host string formats. We have different
+tools that expect the host string to be in different forms. For example,
+`packaging_platform` is specific to how [Vanagon](https://github.com/puppetlabs/vanagon)
+parses that string.
+
+### hosts.yaml
+The rake tasks that run acceptance will by default create a hosts file and
+populate it using [beaker-hostgenerator][] using either the `HOSTS` environment
+variable or the default host string (currently `redhat7-64ma-windows2012r2-64a`).
+The automation assumes you are using the vmpooler hypervisor and an internal
+pooling api, which is only available to Puppet, Inc. employees. If you want to
+customize the hypervisor or the pooling api, you'll need to generate your own
+hosts file. You must pass in a valid host string to the `beaker-hostgenerator`
+command. See the previous section for more information on how to construct a
```

Comment:
nit- just in case we were to re-organize this doc, it's great practice to just name the section, not to state to see 'previous' or 'next'. :)